zip-deployboard-deck/src/snapshots/Snapshots.tsx

```diff
@@ -178,12 +191,58 @@ const SnapshotDeployModal = ({ pipeline, setPipeline, image, onClose, onSubmit }
           </Dropdown.Toggle>
           <Dropdown.Menu>
             {Array.from(SNAPSHOT_CONFIGS.values()).map((config) => (
-              <li>
-                <a onClick={() => setPipeline(config.gitBranch)}>{config.label}</a>
-              </li>
+              <Dropdown.Item>
+                <li>
+                  <a onClick={() => setPipeline(config.gitBranch)}>{config.label}</a>
+                </li>
+              </Dropdown.Item>
             ))}
           </Dropdown.Menu>
         </Dropdown>
+        {pipeline === NEXT_PREVIEW && (
+          <div
+            style={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              paddingTop: '16px',
+            }}
+          >
+            <div style={{ display: 'flex', width: '70%' }}>
+              <Form>
+                <Form.Group className="mb-3" controlId="formBasicEmail">
+                  <Form.Label>Name</Form.Label>
+                  <Form.Control
+                    style={{ maxWidth: '200px' }}
+                    type="name"
+                    placeholder="Enter name"
+                    onChange={(e) => setName(e.target.value.trim())}
+                  />
+                  <Form.Text className="text-muted">You can access this at {name}.next-preview.ziphq.com</Form.Text>
```

Comment:
nit: would be nice if we enclose the link in a <a href="{name}.next-preview.ziphq.com"></a> tag so the user can just clink the link

Reply:
Thats a good idea. Adding!